build.gradle

```diff
@@ -0,0 +1,91 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.15"
```

Comment:
A general Tech stack question: there were some discussion whether to use protobuf, and seems the decision is not for now. 
Is Tanagra not part of McTerra, hence we can choose tech stack on our own? If yes, what is the criteria of choosing them?

Reply:
Nice reference! Thanks!